backend/server/schedule_delete.py

```diff
@@ -0,0 +1,13 @@
+from flask import Blueprint, jsonify
+from tools.delete_slot_by_id_for_schedule import delete_slot_by_id_for_schedule
+
+
+schedule_delete = Blueprint('schedule_delete', __name__)
+
+
+@schedule_delete.route("/schedule/interval_id=<interval_id>", methods=['DELETE'])
+def delete_slots(interval_id):
+    """ Delete slots by id
+    """
+    delete_slot = delete_slot_by_id_for_schedule(interval_id)
```

Comment:
how do you process a missing interval_id? (error code, tests)

Reply:
there is a check in the `backend/tools/delete_slot_by_id_for_schedule.py` file
```py
def delete_slot_by_id_for_schedule(interval_id):
    try:
        models.db.session.delete(Slots.query.get(interval_id))
        models.db.session.commit()
        return {
            'msg': 'successfully deleted'
        }
    except Exception:
        models.db.session.rollback()
        return {
            'msg': 'delete error'
        }
```

Reply:
and test ```py
def test_delete_schedule_slot_if_not_id():
    create_test_app_with_db()
    assert delete_slot_by_id_for_schedule(1) == {'msg': 'delete error'}
```

Reply:
response.status will be 200 even if an error occurred